src/styles/_themes.scss

```diff
@@ -58,6 +58,31 @@
   --form-input-bg: transparent;
   --form-input-text: #{$theme-black};
   --form-input-border: #{color(base-light)};
+
+  // News
+  --btn-box-bg: #{$theme-cosmiclatte-lighter};
+  --btn-box-border: #{$neutral-gray};
+  --date-icon-bg: #{$theme-cosmiclatte-lighter};
+
+  // Tags
+  --source-tag-bg: #{$label-source-lighttheme-bg};
```

Comment:
what do you think about switching the naming on these so `--tag-` is first? then it'll match the rest of the vars, how we have the element itself first and then descriptors? like how you did categories.